bin/cpumask.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+'''Convert CPU list(s) to CPU masks'''
+
+import argparse
+from dataclasses import dataclass
+
+
+@dataclass
+class global_vars:
+    '''Global variables'''
+
+    args = None
```

Comment:
why not just global args in main function?

Reply:
I forget the details...but I think I arrivedat this form of doing it after fighting with pylink on roadblock...more below.